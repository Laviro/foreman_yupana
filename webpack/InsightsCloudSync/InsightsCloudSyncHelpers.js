import { foremanUrl } from '../ForemanRhCloudHelpers';

export const insightsCloudUrl = path => foremanUrl(`/insights_cloud/${path}`);
