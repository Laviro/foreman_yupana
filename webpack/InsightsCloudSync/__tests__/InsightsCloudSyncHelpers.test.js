import { testSelectorsSnapshotWithFixtures } from '@theforeman/test';
import { insightsCloudUrl } from '../InsightsCloudSyncHelpers';

const fixtures = {
  'should return insights cloud Url': () => insightsCloudUrl('test_path'),
};

describe('InsightsCloudSync helpers', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
