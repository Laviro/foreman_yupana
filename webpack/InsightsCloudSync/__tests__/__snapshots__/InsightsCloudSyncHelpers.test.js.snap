// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsightsCloudSync helpers should return insights cloud Url 1`] = `"/insights_cloud/test_path"`;
